packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -134,7 +149,10 @@ export function createOdspNetworkError(
 
     error.online = OnlineStatus[isOnline()];
 
-    const props: ITelemetryProperties = { response: responseText };
+    const facetCodes = responseText !== undefined ? parseFacetCodes(JSON.parse(responseText)) : undefined;
```

Comment:
JSON.parse can throw - it can be anything (including HTML text) - you should protect against it